src/core/instance/events.js

```diff
@@ -19,7 +19,7 @@ export function initEvents (vm: Component) {
   }
 }
 
-let target: Component
+let target: any
```

Comment:
I don't love removing the type annotation here, but if I left it as `Component` then Flow would see the assignment to undefined and error due to "Possible undefined value".

In typescript you can tell the type checker to trust you by doing `target!.method()` but I couldn't find an equivilent in Flow. The Flow docs recommend `if (target !== undefined)`, however I didn't think we'd want to add a branch in the VDom code.

Reply:
What about just binding the `add` and `remove` methods to the `vm` so you don't need this closure variable at all?

Reply:
@chriscasola You won't wanna do it that way. function.bind is just create a closure.